### DISCUSSION TITLE
Issue with Python 3.12 Compatibility and Numba Package Installation

### DISCUSSION THREAD

**Discussion by jongalon** [kitzeslab/opensoundscape#906] [Q&A]
I hope this message finds you well. I wanted to bring to your attention an issue I encountered while attempting to install the opensoundscape software. The recent release of **Python 3.12** has caused an incompatibility with the Numba package, resulting in installation failures.

As Python 3.12 is a new release, the **current version of Numba does not support it**. This causes a hiccup in the installation process, as the package is not compatible with Python 3.12. Consequently, users are unable to install opensoundscape due to this conflict.

I would like to suggest updating the requirements or providing a note to discourage the use of Python 3.12 until Numba is updated to support this new version. It could be beneficial to mention this in the installation instructions or on the project page to prevent similar issues for other users.

Thank you for your attention to this matter. Your work on opensoundscape is highly appreciated. 

Best regards,

Jonathan.

**Comment by sammlapp** [comment]
Thanks for this note. We internally test only on Python 3.9 and 3.10, and we should modify the documentation to specify that these are the "supported" versions as far as internal testing.
